grc/core/params/dtypes.py

```diff
@@ -40,6 +40,11 @@ class ValidateError(Exception):
 def validate_block_id(param,black_listed_ids):
     value = param.value
     # Can python use this as a variable?
+
+    is_exempt_from_id_validation = hasattr(param.parent_block, 'exempt_from_id_validation')
```

Comment:
```suggestion
    is_exempt_from_id_validation = getattr(param.parent_block, 'exempt_from_id_validation', False)
```

Reply:
Goodness me. How did I not catch that? Thanks! Will make the amendment